### ISSUE TITLE
External brim or raft for small support trees to aid adhesion

### ISSUE THREAD

**Issue by molove** [Ultimaker/Cura#11028]
### Is your feature request related to a problem?

I really like tree supports especially from the build plate, often I have a piece that only needs small support trees. I try not to use brims on pieces that don't need them. Unless I use a brim, often small support trees don't stick to the print bed.

For instance something like this would probably not stick to the bed

![image](https://user-images.githubusercontent.com/20013346/145035582-037b37f7-beae-4060-b0b9-a6b54cda098e.png)

### Describe the solution you'd like

I'm not sure if this has been asked for before but I would like the option to add an external brim or raft to support trees only to aid bed adhesion and not to the piece as well. Or perhaps just to support trees whose bases are less than a certain area.

### Describe alternatives you've considered

I've tried the internal brim for supports but it doesn't help. Using a brim for the model does work but isn't an ideal solution as it adds unecessary clean up to the model.

### Affected users and/or printers

I think it would be beneficial to everyone.

**Comment by fvrmr** [comment]
Hi @molove thank you for your feature request. 
I have discussed this with an colleague and we believe this a edge case and this is not a feature with high priority for us to add right now. That's why I will defer this request.

However there is a workaround that can help you. You can use the `brim distance` setting for the normal brim, this will create a gap between the brim and the model and this will not affect the brim for the support. So the brim will be attached to the support but not the model.

**Comment by Phusck** [comment]
WOuld really like to see this

**Comment by onolox** [comment]
This feature is REALLY needed, it's not a "edge case".
Trees really need a bigger base to stay in the bed.

**Comment by CWempe** [comment]
I too think it would make sense to add a brim option for supports only.

Until then, I add a little info to the workaround, because only setting the "Brim Distance" is not the best/cleanest option for me.

Without brim:
![image](https://github.com/Ultimaker/Cura/assets/1270359/96b63d4d-ece9-46ff-9090-5bb57a4bef80)



With brim:
![image](https://github.com/Ultimaker/Cura/assets/1270359/649c9b0c-2514-422c-aff9-989c9d96cfa5)


With 2 mm brim distance:
![image](https://github.com/Ultimaker/Cura/assets/1270359/9501471f-a452-4b68-af1a-b0623cb366bf)

With 2 mm "brim distance" and 0 mm "Brim Width": 👎
![image](https://github.com/Ultimaker/Cura/assets/1270359/01535e38-c37c-427a-9337-092c419a3355)

With 2 mm "brim distance" and 1 "Brim Line Count": 👍
![image](https://github.com/Ultimaker/Cura/assets/1270359/834f86d8-9885-4f1a-9689-49763e18aef5)

So you cannot get rid of the last Brim Line, but you can set a bigger distance.
Unfortunately the distance is valid for support trees. too.-
But at least there is a brim inside of the tree.

**Comment by klwemu** [comment]
Tree Support does not stick enough. And brim distance is not a working solution. 
Tree support with brim is in PrusaSlicer and OrcaSlicer a pretty cool default feature.

**Comment by Quedale** [comment]
I also agree that such feature would be really useful.
The higher a tree support gets, without touching the model with a branch, it becomes really weak.
Tall(ish) trees only need a very slight drag from the nozzle/filament to pull the entire tree from the plate.

The only workaround I found is to use a raft, which may defeat the purpose of using tree support in the first place depending on the model.

**Comment by adamsthws** [comment]
This is something I'd love to see in Cura too!

**Comment by njsokalski** [comment]
I agree that this would be extremely useful and advantageous, I have had tons of Tree Supports collapse because they got too tall without enough adhesion, and am yet to find a good solution.
